packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -60,12 +61,26 @@ class ConfigScreenComponent extends React.Component<any, any> {
 		this.setState({ settings: this.props.settings });
 	}
 
+	updatePluginsStates(value: any) {
+		const key = 'plugins.states';
+		const md = Setting.settingMetadata(key);
+		shared.updateSettingValue(this, key, value);
+
+		if (md.autoSave) {
+			shared.scheduleSaveSettings(this);
+		}
+	}
+
 	componentDidMount() {
 		if (this.props.defaultSection) {
 			this.setState({ selectedSectionName: this.props.defaultSection }, () => {
 				this.switchSection(this.props.defaultSection);
 			});
 		}
+		if (!Setting.value('updatedDefaultPluginsInstallStates')) {
```

Comment:
Doesn't it mean the plugin settings will be set only once? How about when we add a new default plugin?

Reply:
Added `installedDefaultPlugins` array to check if new default plugin is available.